packages/ui/src/theme/tokens/components/inAppMessaging/backdrop.ts

```diff
@@ -0,0 +1,9 @@
+import { BackgroundColorValue, DesignToken } from '../../types/designToken';
+
+export interface BackdropTokens {
+  backgroundColor: DesignToken<BackgroundColorValue>;
```

Comment:
What about nesting the different components inside of InAppMessagingTokens instead?
```suggestion
  backdrop: {
     backgroundColor: DesignToken<BackgroundColorValue>;
  }
```